plugin/src/main/resources/data/abilities.yml

```
@@ -62,4 +62,23 @@ abilities:
+    displayItem: ender_pearl
+    metadata:
+      teleportDistance: 16.0
+
+  - id: block_toss
+    cooldown: 2.5
+    itemSlot: 0
+    type: projectile
+    usage: right_click
+    hotbarItem: iron_sword
+    displayItem: grass_block
+    metadata:
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)